charts/microgateway/values.yaml

```diff
@@ -97,7 +97,7 @@ config:
   # config.generic -- Available for:<br>
   # - [Simple DSL configuration](#simple-dsl-configuration)<br>
   # - [Advanced DSL configuration](#advanced-dsl-configuration)<br>
-  # - [Expert DSL configuration](#expert-dsl-configuration)
+  # - [Expert DSL configuration](#expert-dsl-configuration).
```

Comment:
Why does Expert DSL configuration have a dot. 
And the others not?

Reply:
I removed the ".".